examples/pnp_demo/src/main.rs

```diff
@@ -95,7 +95,17 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .collect();
 
     // Run EPnP
-    let result = kpnp::solve_pnp(&world_pts, &image_pts, &k, kpnp::PnPMethod::EPnPDefault)?;
+    let distortion = kpnp::PolynomialDistortion {
```

Comment:
how can we verify correctness here; can we provide  any image with that distiortion ?